benchmarks/benchmarks/bench_ufunc.py

```diff
@@ -81,6 +81,8 @@ def setup(self):
         self.i = np.ones(150000, dtype=np.int32)
         self.f = np.zeros(150000, dtype=np.float32)
         self.d = np.zeros(75000, dtype=np.float64)
+        
+        self.tf = np.ones(150000, dtype=np.int32)
```

Comment:
Isn't this the same as `self.i`, so you can reuse that instead?

Reply:
Just to point it out, but you do not want to use an integer array in the test.  This will trigger casting.  You need to use a homogeneous signature (i.e. both inputs are float32 or float64).
The speedup may still not be huge (this is probably memory-bound anyway), but adding a cast might hide most of it even if it is huge.

Reply:
Hi @rgommers, @seberg do you guys know how can I run two benchmark for comparing the original implementation and the SIMD implementation? I have changed the benchmark test for using float64, but the elapsed time increased. I think it is quite weird, and the 10% speedup is weird for my side too. I thought it will accelerate much more.

Reply:
There's a missing understanding here, this patch only adds universal intrinsics for copysign but without actually using them in the inner loop of ufunc so there's no need for a benchmark.